templates/hello-world/server.ts

```diff
@@ -26,7 +25,7 @@ export default {
         throw new Error('SESSION_SECRET environment variable is not set');
       }
 
-      const waitUntil = (p: Promise<any>) => executionContext.waitUntil(p);
+      const waitUntil = executionContext.waitUntil.bind(executionContext);
```

Comment:
FYI I'm changing these lines to use `bind` because that's what we have in our `skeleton` template. It's not as easy to understand as arrow functions but it keeps the type correct in JS projects, which I think is useful?